data/json/mutations/mutations.json

```diff
@@ -1831,7 +1867,7 @@
     "visibility": 5,
     "ugliness": 3,
     "description": "You've grown a set of gills in your neck, allowing you to breathe underwater.  Slightly increases wet benefits.",
-    "category": [ "FISH" ],
+    "category": [ "FISH", "BATRACHIAN" ],
```

Comment:
From the tadpole stage?

Reply:
I think there might be a different trait that would make this superfluous. But I'd need to look for it specifically.

Reply:
OK so I'm thinking I see what you mean, I should ditch the gills and create a unique mutation that has the same impact as gills but comes from the osmatic skin type stuff?

Reply:
Ah here it is  Gills is a prerequisite to getting  
``` {
    "type": "mutation",
    "id": "FRESHWATEROSMOSIS",
    "name": { "str": "Freshwater Osmosis" },
    "points": 4,
    "description": "Your gills and skin are highly permeable to moisture, and you can hydrate your body through osmosis.  While submerged in fresh water, any thirst you might have will be slaked.",
    "prereqs": [ "GILLS" ],
    "category": [ "FISH", "BATRACHIAN" ],
    "threshreq": [ "THRESH_FISH", "THRESH_BATRACHIAN" ]
  },
```